nemo_automodel/components/moe/layers.py

```diff
@@ -660,12 +670,12 @@ def forward(
             indices = torch.topk(scores, self.topk, dim=-1)[1]
             weights = original_scores.gather(1, indices)
 
-            if self.score_func == "sigmoid" and self.norm_topk_prob and self.topk > 1:
-                # Use out-of-place division to keep autograd versions intact.
-                denom_w = weights.sum(dim=-1, keepdim=True) + 1e-20
-                denom_s = original_scores.sum(dim=-1, keepdim=True) + 1e-20
-                weights = weights / denom_w
-                original_scores = original_scores / denom_s
+        if self.norm_topk_prob and self.topk > 1:
+            # Use out-of-place division to keep autograd versions intact.
+            denom_w = weights.sum(dim=-1, keepdim=True) + 1e-20
+            denom_s = original_scores.sum(dim=-1, keepdim=True) + 1e-20
+            weights = weights / denom_w
+            original_scores = original_scores / denom_s
```

Comment:
seems like this will now be applied in the softmax case too after this change. Is this intended?